distributed/tests/test_scheduler.py

```diff
@@ -2763,13 +2763,16 @@ async def test_get_task_duration(c, s, a, b):
 
 @gen_cluster(client=True)
 async def test_default_task_duration_splits(c, s, a, b):
-    """Ensure that the default task durations for shuffle split tasks are, by default,
+    """Ensure that the default task durations for shuffle split tasks are
     aligned with the task names of dask.dask
     """
     pd = pytest.importorskip("pandas")
     dd = pytest.importorskip("dask.dataframe")
 
-    # We don't care about the actual computation here but we'll schedule one anyhow to
+    if dd._dask_expr_enabled():
+        pytest.skip("No split task in dask-expr")
```

Comment:
@fjetter @phofl can I have a secondo opinion on this? Not sure I understand why we don't have this task in dask-expr. Are we missing a line in `distributed.scheduler.default-task-durations`?

Reply:
Same discussion for `test_blocklist_shuffle_split`

Reply:
@hendrikmakait and I stumbled upon the root cause, dask-expr doesn't properly respect the context manager. I am preparing a fix at the moment, should be there in a minute

Reply:
https://github.com/dask-contrib/dask-expr/pull/958

Reply:
pr is merged

Reply:
@phofl I'm now seeing `split-taskshuffle` and `split-stage`. Are they both supposed to have 1us default-duration?

Reply:
Yeah the naming is confusing and something we probably should fix, but it's ok conceptually